assets/src/blocks/CarouselHeader/EditableBackground.js

```diff
@@ -1,11 +1,13 @@
 import { MediaUpload, MediaUploadCheck } from '@wordpress/block-editor';
 import { ImagePlaceholder } from './ImagePlaceholder';
 import { Button, Dropdown } from '@wordpress/components';
+import { toSrcSet } from '../../../../../../themes/planet4-master-theme/assets/src/js/Components/archivePicker/sizeFunctions';
```

Comment:
Isn't it a bit weird to store this function in the `archivePicker` folder? They're not related 🤔

Reply:
Yes, still need to move it.

Reply:
I ended up copying the function, as I just overlooked that it was not even in the same repo. It's a simple function and we need to make adjustments because the data for `select('core').getMedia(id)` uses different keys than what is returned in `<MediaUpload onSelect/>` :facepalm: